src/components/fields/SchemaField.js

```diff
@@ -102,17 +102,19 @@ function ErrorList(props) {
   if (errors.length === 0) {
     return <div />;
   }
+
   return (
     <div>
-      <p />
```

Comment:
Was it intentional or just a leftover?

Reply:
Not sure, do you know what commit caused this deletion?

Reply:
I mean I deleted this `<p />`, but I'm kinda curious why was it here in the first place 😛

Reply:
Probably an accident then